src/components/ForestTypeDescription/lu/Site.js

```diff
@@ -10,287 +10,1194 @@ function Site({ data }) {
+                  <text
+                    x="43.443"
+                    y="-6.713"
+                    style={{
+                      lineHeight: '1.25',
+                      InkscapeFontSpecification: "'sans-serif, Normal'",
+                      fontVariantLigatures: 'normal',
+                      fontVariantCaps: 'normal',
+                      fontVariantNumeric: 'normal',
+                      fontFeatureSettings: 'normal',
+                      WebkitTextAlign: 'start',
+                      textAlign: 'start',
+                    }}
+                    fill="#000"
+                    fillOpacity="1"
+                    stroke="none"
+                    display="inline"
+                    fontFamily="sans-serif"
+                    fontSize="8.819"
+                    fontStretch="normal"
+                    fontStyle="normal"
+                    fontVariant="normal"
+                    fontWeight="normal"
+                    letterSpacing="0"
+                    textAnchor="start"
+                    wordSpacing="0"
+                    writingMode="lr-tb"
+                    xmlSpace="preserve"
+                  >
+                    <tspan x="43.443" y="-6.713">
+                      S
+                    </tspan>
+                  </text>
+                  <text
+                    x="79.682"
+                    y="-43.198"
+                    style={{
+                      lineHeight: '1.25',
+                      InkscapeFontSpecification: "'sans-serif, Normal'",
+                      fontVariantLigatures: 'normal',
+                      fontVariantCaps: 'normal',
+                      fontVariantNumeric: 'normal',
+                      fontFeatureSettings: 'normal',
+                      WebkitTextAlign: 'start',
+                      textAlign: 'start',
+                    }}
+                    fill="#000"
+                    fillOpacity="1"
+                    stroke="none"
+                    display="inline"
+                    fontFamily="sans-serif"
+                    fontSize="8.819"
+                    fontStretch="normal"
+                    fontStyle="normal"
+                    fontVariant="normal"
+                    fontWeight="normal"
+                    letterSpacing="0"
+                    textAnchor="start"
+                    wordSpacing="0"
+                    writingMode="lr-tb"
+                    xmlSpace="preserve"
+                  >
+                    <tspan x="79.682" y="-43.198">
+                      O
+                    </tspan>
+                  </text>
+                </g>
+              </g>
             </g>
           </g>
         </g>
         {!data ||
           (data.includes(null) && (
-            <NoData
-              height={100}
-              width={120}
-              className={styles.labelSmallBold}
-            />
+            <NoData height={80} width={85} className={styles.labelXsBold} />
```

Comment:
To me it looks like `NoData` doesn't support the `className` prop.

Reply:
It does: https://github.com/geops/tree-app/pull/538/files#diff-2d8775ada217168787531cc9c8ad7620bab224ebf9645be954832b637e3115f1R7

Reply:
Ah, sorry, only looked at master.

LGTM